File: task3/Elixir/Subtask2.exs
# Define a module with a function to perform file copy
defmodule FileCopy do
  def copy_contents(input_file, output_file) do
    case File.read(input_file) do
      {:ok, content} ->
        case File.write(output_file, content) do
          :ok -> IO.puts("Content successfully copied to #{output_file}.")
          {:error, reason} -> IO.puts("Failed to write to output file: #{reason}")
        end
      {:error, reason} -> IO.puts("Failed to read from input file: #{reason}")
    end
  end
end

# File names
input_file = "input.txt"
output_file = "output.txt"

# Perform the file copy
FileCopy.copy_contents(input_file, output_file)

File: task3/Ruby/Subtask2.rb
# Define file names
input_file = 'input.txt'
output_file = 'output.txt'

begin
  # Read the content from input.txt
  content = File.read(input_file)

  # Write the content to output.txt
  File.write(output_file, content)

  puts "Content successfully copied from #{input_file} to #{output_file}."
rescue Errno::ENOENT => e
  # Handle the case where the file does not exist
  puts "Error: #{e.message}"
rescue IOError => e
  # Handle other I/O related errors
  puts "Error: #{e.message}"
end
